lib/backup/s3remote/s3.go

```diff
@@ -99,9 +99,14 @@ func (fs *FS) Init() error {
 		config.WithDefaultRegion("us-east-1"),
 	}
 
-	if len(fs.CredsFilePath) > 0 {
+	if len(fs.ConfigFilePath) > 0 {
 		configOpts = append(configOpts, config.WithSharedConfigFiles([]string{
 			fs.ConfigFilePath,
+		}))
+	}
+
+	if len(fs.CredsFilePath) > 0 {
+		configOpts = append(configOpts, config.WithSharedCredentialsFiles([]string{
```

Comment:
Based on AWS SDK code here - https://github.com/VictoriaMetrics/VictoriaMetrics/blob/b12864f9055afa4abd5b8c4f2fe775b55e24402d/vendor/github.com/aws/aws-sdk-go-v2/config/load_options.go#L386-L391
It looks like it is required to construct an array of files an pass an array only once as otherwise it will override the value. 
Could you please update the PR to address this?

Reply:
WithSharedConfigFiles is executed only once

Reply:
Ah, sorry, my bad!